lms/static/sass/discussion/_discussion-v1.scss

```diff
@@ -41,21 +41,17 @@
     &:hover,
     &:active,
     &:focus {
-        border-color: $forum-color-hover;
-        background-color: $forum-color-hover;
-        background-image: none;
-        box-shadow: none;
-        color: $forum-color-active-text;
-        text-decoration: none;  // Don't show underlines on links that are styled as buttons
+        border-color: $forum-color-hover !important;
+        background-color: $forum-color-hover !important;
+        background-image: none !important;
+        box-shadow: none !important;
+        color: $forum-color-active-text !important;
+        text-decoration: none !important;  // Don't show underlines on links that are styled as buttons
```

Comment:
`!important` is 😓, in cases where it's really neccessary what I've been doing is adding a comment that links to the line this one is overriding, just so people know why it was added.